<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>347</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>632</width>
    <height>347</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>632</width>
    <height>347</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DevirtaPics. v.0.0.1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Режимы</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="testing_tab">
          <attribute name="title">
           <string>Тестирование</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="frameShape">
              <enum>QFrame::WinPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="text">
              <string>Описание: Режим тестирования предназначен для определения преобладающего типа дыхания у пациента. </string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Время проведения процедуры (мин):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTimeEdit" name="testing_time">
               <property name="maximumTime">
                <time>
                 <hour>1</hour>
                 <minute>59</minute>
                 <second>0</second>
                </time>
               </property>
               <property name="minimumTime">
                <time>
                 <hour>0</hour>
                 <minute>1</minute>
                 <second>0</second>
                </time>
               </property>
               <property name="displayFormat">
                <string>mm</string>
               </property>
               <property name="calendarPopup">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Ваш результат:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="domin_bt_val">
               <property name="text">
                <string>ЗНАЧЕНИЕ НЕ ОПРЕДЕЛЕНО</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="start_test_btn">
               <property name="text">
                <string> Запустить режим </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="rehab_tab">
          <attribute name="title">
           <string>Реабилитация</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="frameShape">
              <enum>QFrame::WinPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="text">
              <string>Описание: Режим реабилитации позволяет отслеживать типы дыхания у пациентов для процесса реабилитации (изменения) типа дыхания пациента.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Брюшное:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="stom_br_val">
                 <property name="text">
                  <string>НЕ ОПРЕДЕЛЕНО</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Грудное:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="chest_br_val">
                 <property name="text">
                  <string>НЕ ОПРЕДЕЛЕНО</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Смешанное:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="mix_br_val">
                 <property name="text">
                  <string>НЕ ОПРЕДЕЛЕНО</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="start_rehab_btn">
               <property name="text">
                <string> Запустить режим </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="back_btn">
        <property name="text">
         <string>Назад</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="settings">
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QMenu" name="camera_s">
     <property name="title">
      <string>Камера</string>
     </property>
     <addaction name="check_cam"/>
     <addaction name="restart_cam"/>
    </widget>
    <widget class="QMenu" name="detection_s">
     <property name="title">
      <string>Настройки распознавания</string>
     </property>
     <addaction name="graph_s"/>
     <addaction name="analyser_s"/>
    </widget>
    <addaction name="camera_s"/>
    <addaction name="detection_s"/>
   </widget>
   <addaction name="settings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <action name="action">
   <property name="text">
    <string>Проверить камеру</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Перезагрузить камеру</string>
   </property>
  </action>
  <action name="check_cam">
   <property name="text">
    <string>Проверить камеру</string>
   </property>
  </action>
  <action name="restart_cam">
   <property name="text">
    <string>Переподключить камеру</string>
   </property>
  </action>
  <action name="graph_s">
   <property name="text">
    <string>Настройки графика</string>
   </property>
  </action>
  <action name="analyser_s">
   <property name="text">
    <string>Настройки анализатора</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
